xAPI-Communication.md

```diff
@@ -726,13 +730,13 @@ are known with reasonable certainty to be available for retrieval. This time SHO
 account any temporary condition, such as excessive load, which might cause a delay in Statements 
 becoming available for retrieval.
 
-* If the attachment property of a GET statement is used and is set to <code>true</code>, the LRS MUST use the 
+* If the "attachment" property of a GET statement is used and is set to <code>true</code>, the LRS MUST use the 
 multipart response format and include all attachments as described in <a href="#attachments">4.1.11</a>.
 
-* If the attachment property of a GET statement is used and is set to <code>false</code>, the LRS MUST NOT
+* If the "attachment" property of a GET statement is used and is set to <code>false</code>, the LRS MUST NOT
 include attachment raw data and MUST report application/json.
 
-* The LRS SHOULD* include a "Last-Modified" header which matches the Stored timestamp of the Statement. 
+* The LRS SHOULD* include a "Last-Modified" header which matches the "Stored" timestamp of the Statement. 
```

Comment:
"stored" is going to wreck our quotes usage, because quotes are necessary in many of the cases to ensure people don't think it is an adjective.  Thanks for spotting this error.